website/docs/foundations/elevation/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Elevation
+caption: MD - “Elevation is the relative distance between two surfaces along the z-axis.”
 ---
```

Comment:
Copied from Material Design just as an example. Do we want something similar to this?